### DISCUSSION TITLE
eForms SDK 1.3.0

### DISCUSSION THREAD

**Discussion by bertrand-lorentz** [OP-TED/eForms-SDK#190] [Announcements]
We have released version 1.3.0 of the [eForms Software Development Kit](https://github.com/OP-TED/eForms-SDK/releases/tag/1.3.0).

This release brings various changes and corrections in the metadata content.

Please see the [release notes](https://github.com/OP-TED/eForms-SDK/releases/tag/1.2.0) for more details.

The release artifact is available for [download in Maven Central](https://search.maven.org/search?q=g:%22eu.europa.ted.eforms%22%20AND%20a:%22eforms-sdk%22). It might take a few hours for the new version to appear in searches.

**Comment by DeMol-EE** [comment]
Hello and thanks for this release.
With the 1.3.0 files, our EFX transpiler/preprocessor is running into an issue: notice-subtype 20 is referencing field with id 'BT-762-notice', but this field no longer seems to exist in fields.json (it was there in 1.2.1). This is just one example and I suspect there could be more instances, but our procedure stops at the first error.
Could you please take a look and verify if the metadata is broken?
Thanks again.

**Reply by bertrand-lorentz** [reply]
Thanks for the feedback.
There are indeed 2 fields that are missing from fields.json : "BT-762-notice" and "BT-803(t)-notice".
They were incorrectly omitted due to an issue in the generation of the file.

The correction is already available in the "develop" branch, and will be included in SDK 1.3.1, which should be released in the next few days.

**Reply by bertrand-lorentz** [reply]
SDK 1.3.1 has now been released: https://github.com/OP-TED/eForms-SDK/releases/tag/1.3.1

**Reply by jpmckinney** [reply]
BT-271-Part and OPT-090-LotsGroup were also removed. OPT-090-LotsGroup was removed following https://github.com/OP-TED/eForms-SDK/discussions/135 Just checking that removal of BT-271-Part was intentional.

**Reply by YvesJo** [reply]
Indeed, removal of BT-271-Part is intentional as no Framework Agreement information shall actually be expected for PIN Only notices.

**Comment by odscjen** [comment]
Hi,
Some of the [XML examples in the SDK documentation](https://docs.ted.europa.eu/eforms/1.3.0/schema/all-in-one.html#_notice_aggregated_amounts) don't appear to have all been updated to reflect the changes in 1.3.0. For example BT-118 has changed from `EstimatedOverallFrameworkContractsAmount` to `OverallMaximumFrameworkContractsAmount` but the XML example still contains `EstimatedOverallFrameworkContractsAmount`.

**Reply by YvesJo** [reply]
Thanks for pointing this out, this will get fixed in a next documentation release.

**Comment by bertrand-lorentz** [comment]
eForms SDK 1.3.2 has been released:
https://github.com/OP-TED/eForms-SDK/releases/tag/1.3.2

The documentation website has also been updated.
